docs/changelog.rst

```diff
@@ -2,7 +2,14 @@
 Changelog
 =========
 
+<<<<<<< HEAD
```

Comment:
you left the merge markers in here. Can you remove them?

It's the `<<<<<<< HEAD` and  `=======` and `>>>>>>> Add support for UMA token`.

Other than that looks good :)